### PR TITLE
changing prod dataset

### PR DESCRIPTION
**Pull request by mhigbyflowers**
Using the new CDR hotfix

### CODE REVIEW

File: public-api/config/cdr_versions_prod.json [hunk 1]
```diff
@@ -27,10 +27,10 @@
     "name": "All of Us Dataset v2",
     "dataAccessLevel": 1,
     "bigqueryProject": "aou-res-curation-prod",
-    "bigqueryDataset": "combined20191004_dbrowser_1",
+    "bigqueryDataset": "combined20191004_dbrowser_2",
```
Comment (mhigbyflowers) [inline public-api/config/cdr_versions_prod.json:30]: new hotfix

File: public-api/config/cdr_versions_prod.json [hunk 2]
```diff
@@ -27,10 +27,10 @@
...
     "creationTime": "2020-02-11 16:27:20Z",
     "releaseNumber": 2,
     "numParticipants": 225360,
-    "publicDbName": "p_2019q1_33"
+    "publicDbName": "p_2020q2_1"
```
Comment (mhigbyflowers) [inline public-api/config/cdr_versions_prod.json:34]: updating with the correct naming convention.

File: public-api/config/cdr_versions_prod.json [hunk 3]
```diff
@@ -27,10 +27,10 @@
     "name": "All of Us Dataset v2",
     "dataAccessLevel": 1,
     "bigqueryProject": "aou-res-curation-prod",
-    "bigqueryDataset": "combined20191004_dbrowser_1",
+    "bigqueryDataset": "combined20191004_dbrowser_2",
     "creationTime": "2020-02-11 16:27:20Z",
     "releaseNumber": 2,
     "numParticipants": 225360,
```
Comment (freemabd) [inline public-api/config/cdr_versions_prod.json]: Should you update the participant count since participants have been removed?

**Review by freemabd — APPROVED** [review]
Looks good, just one question

File: public-api/config/cdr_versions_prod.json (hunk 3 above)
Comment (mhigbyflowers) [reply public-api/config/cdr_versions_prod.json]: thanks for catching that. I have updated value

**Pull request merged**